### DISCUSSION TITLE
image active selecttion render wrong?

### DISCUSSION THREAD

**Discussion by sanheng** [fabricjs/fabric.js#7272] [Q&A]
Read the svg, the display is not correct when the picture is activated

![image](https://user-images.githubusercontent.com/13793256/128156117-3a15afa7-a8b1-48a0-aa03-f4d5d55ba0bb.png)

**Comment by asturur** [comment]
hi @sanheng please follow the instructions for issues.
- include the svg
- load it in a fiddle so that we can check

**Reply by sanheng** [reply]
It is correct in codesandbox, but it is wrong in my local, I will check again
